frontend/src/components/board/footer/Reaction.tsx

```diff
@@ -0,0 +1,116 @@
+import { makeStyles } from "@material-ui/core";
+import React, { useEffect } from "react";
+
+const useStyles = makeStyles({
```

Comment:
Since this styling block is really big we could think of possible solutions. 

1. Extract the styles to its own file and import it here
2. Change the order. So component definition stays at the top and styles at the bottom

The reason for that is simply DX. Personally, I wouldn't want to scroll so much in order to see the actual component. I hope this makes sense somehow.

Reply:
I put those styles at the end of the file. That made absolutely sense!